holidays/countries/india.py

```
@@ -199,7 +199,7 @@ def _populate_public_holidays(self):
         # Hindu Holidays.
         if self._year < 2001 or self._year > 2035:
             warning_msg = "Requested Holidays are available only from 2001 to 2035."
-            warnings.warn(warning_msg, Warning)
+            warnings.warn(warning_msg, UserWarning)
```

Ruff (0.15.10)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)